### PR TITLE
Templates for issue/bug-report added

### LINKED ISSUE #8: Create an Issue template for the project.
We need to create an issue template for this project which would be great for maintainers and contributers. I would like to take up this task.

### PR DESCRIPTION
**Pull request by ritvij14**
## Description:
Added templates for creating issues, for feature requests and bug reports

## Related Issues:
Fixes #8

### CODE REVIEW

**Comment by ritvij14** [comment]
@KeenWarrior @GangaChatrvedi @Abhishek-kumar09 please review this.

File: .github/ISSUE_TEMPLATES/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```
Comment (Abhishek-kumar09) [inline .github/ISSUE_TEMPLATES/bug_report.md:9]: Also add:
`<!-- Remove anything that is not applying to your issue/bug context -->`

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

**Comment by Abhishek-kumar09** [comment]
Rename Folder
`ISSUE_TEMPLATES` to `ISSUE_TEMPLATE`

File: .github/ISSUE_TEMPLATES/bug_report.md (hunk 1 above)
Comment (ritvij14) [reply .github/ISSUE_TEMPLATES/bug_report.md:9]: @Abhishek-kumar09 done

**Comment by ritvij14** [comment]
> Rename Folder
> `ISSUE_TEMPLATES` to `ISSUE_TEMPLATE`

@Abhishek-kumar09 done

**Pull request merged**